dagger-core/src/main/java/com/gotocompany/dagger/core/processors/external/grpc/GrpcSourceConfig.java

```diff
@@ -24,6 +24,8 @@ public class GrpcSourceConfig implements Serializable, SourceConfig {
     private String grpcMethodUrl;
     private String requestPattern;
     private String requestVariables;
+    private String grpcArgKeepaliveTimeMs;
+    private String grpcArgKeepaliveTimeoutMs;
```

Comment:
can you rename these two to be - 
```suggestion
    private String grpcArgKeepAliveTimeMs;
    private String grpcArgKeepAliveTimeoutMs;
```

Reply:
keepalive is a single word in grpc, if we change to grpcArgKeepAliveTimeMs, in the configuration we need to pass grpc_arg_keep_alive_time_ms
https://grpc.io/docs/guides/keepalive/